clientd3d/d3drender_bgoverlays.c

```diff
@@ -142,13 +138,13 @@ void D3DRenderBackgroundOverlays(const BackgroundOverlaysRenderStateParams& bgoR
 		static const float u[4] = { epsilon, 1.0f - epsilon, 1.0f - epsilon, epsilon };
 		static const float v[4] = { epsilon, epsilon, 1.0f - epsilon, 1.0f - epsilon };
 
-		pChunk->st0[0].s = u[0];
+		pChunk->st0[0].s = u[1];
```

Comment:
Can you add a comment here explaining why the texture coordinates aren't in the expected order?